.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - node
+  - lts/*
```

Comment:
Why?

Reply:
Because native codes from "jsonld-signatures" cannot build with the latest
stable Node.js in Travis CI, so use the latest LTS Node.js instead.

Reply:
You have to explain explicitly in Git commit messages, otherwise it is almost impossible to track.

Reply:
Got it, I'll try to improve commit message.

Reply:
Commit message updated.